src/lib.rs

```diff
@@ -1,13 +1,31 @@
-//! # num_cpus
-//!
 //! A crate with utilities to determine the number of CPUs available on the
 //! current system.
+//! 
+//! Sometimes the cpu will exaggerate the number of cpus it contains, because it can use
```

Comment:
I think it would be a good idea to stick to writing "CPU" or "CPUs" in capital letters, for consistency.